src/main/scala/rise/core/package.scala

```diff
@@ -40,4 +42,8 @@ package object core {
       case p: Primitive        => p.toString
     }
   }
+
+  implicit def primitiveBuilderToPrimitive(pb: Builder
```

Comment:
maybe `Builder` should be `PrimitiveBuilder`?

Reply:
That was the name I had before. I also thought of making it `Primitive.Builder`.
Opinions?